airflow/cli/commands/pool.py

```diff
@@ -0,0 +1,162 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Pools sub-commands"""
+import json
+import os
+from json import JSONDecodeError
+
+import rich_click as click
+
+from airflow.api.client import get_current_api_client
+from airflow.cli import airflow_cmd, click_output, click_verbose
+from airflow.cli.simple_table import AirflowConsole
+from airflow.exceptions import PoolNotFound
+from airflow.utils.cli import suppress_logs_and_warning_click_compatible
+
+
+@airflow_cmd.group("pools")
+def pools():
+    """Manage pools"""
+
+
+def _show_pools(pools, output):
+    AirflowConsole().print_as(
+        data=pools,
+        output=output,
+        mapper=lambda x: {
+            "pool": x[0],
+            "slots": x[1],
+            "description": x[2],
+        },
+    )
+
+
+@pools.command("list")
+@click_output
+@click_verbose
+@suppress_logs_and_warning_click_compatible
+def pool_list(output, verbose):
+    """Displays info of all the pools"""
+    api_client = get_current_api_client()
+    pools = api_client.get_pools()
+    _show_pools(pools=pools, output=output)
+
+
+@pools.command("get")
+@click.argument("name")
+@click_output
+@click_verbose
+@suppress_logs_and_warning_click_compatible
+def pool_get(name, output, verbose):
+    """Displays pool info by a given name"""
+    api_client = get_current_api_client()
+    try:
+        pools = [api_client.get_pool(name=name)]
+        _show_pools(pools=pools, output=output)
+    except PoolNotFound:
+        raise SystemExit(f"Pool {name} does not exist")
+
+
+@pools.command("set")
+@click.argument("name")
+@click.argument("slots")
+@click.argument("description")
+@click_verbose
+@suppress_logs_and_warning_click_compatible
+def pool_set(name, slots, description, verbose):
+    """Creates new pool with a given name and slots"""
+    api_client = get_current_api_client()
+    api_client.create_pool(name=name, slots=slots, description=description)
+    print(f"Pool {name} created")
```

Comment:
I'd like to try to route all printed output from commands through Rich's Console object, if possible:

```suggestion
    from rich.console import Console
    console = Console()
    ...
    console.print(f"Pool {name} created")
```

Reply:
Missed that, thanks.